plugins/pulp_rpm/plugins/importers/yum/associate.py

```diff
@@ -188,10 +194,10 @@ def filter_available_rpms(rpms, import_conduit, repo):
                                        models.RPM, repo)
 
 
-def copy_rpms(units, source_repo, dest_repo, import_conduit, config, copy_deps, solver=None):
+def copy_rpms(units, source_repo, dest_repo, import_conduit, config, solver=None):
     """
     Copy RPMs from the source repo to the destination repo, and optionally copy
```

Comment:
I think the bit behind the comma, "and optionally copy dependencies as well", should also be removed.